docs/examples/workflows-examples.md

```diff
@@ -107,6 +107,7 @@ Explore the examples through the side bar!
 | [template-defaults](https://github.com/argoproj/argo-workflows/blob/master/examples/template-defaults.yaml) |
 | [testvolume](https://github.com/argoproj/argo-workflows/blob/master/examples/testvolume.yaml) |
 | [timeouts-step](https://github.com/argoproj/argo-workflows/blob/master/examples/timeouts-step.yaml) |
+| [webhdfs-input-output-artifacts](https://github.com/argoproj/argo-workflows/blob/master/examples/webhdfs-input-output-artifacts.yaml) |
```

Comment:
FYI upstream example contains an extra field that is not accepted, so moved to hera examples. I looked at the OpenAPI spec on main and it does not have the field so moved until upstream is fixed